src/main/java/picard/analysis/CollectAlignmentSummaryMetrics.java

```diff
@@ -119,18 +119,25 @@
     @Argument(shortName="BS", doc="Whether the SAM or BAM file consists of bisulfite sequenced reads.")
     public boolean IS_BISULFITE_SEQUENCED = false;
 
+    @Argument(doc = "Avoid collecting actual alignment information. Only count READS, PF_READS, and NOISE_READS. (For backwards compatibility).")
+    public boolean DO_NOT_COLLECT_ALIGNMENT_INFORMATION = false;
```

Comment:
Not sure if there's a picard convention on this, but I think it's easier to work with boolean flags if they're stated in the positive sense. E.g. this would instead be `COLLECT_ALIGNMENT_INFORMATION` and it would default to `true`.

Reply:
:+1: